### PR TITLE
Artik-151: Fix start workspace from on any stack

### PR DESCRIPTION
**Pull request by vparfonov**
On start ws-agent will check is directory with Artik API Docs exist. 
If it exist special property will be set to the location of this folder
if not map it to the webapps/ROOT. We need this workaround for staring WS-Agent in any case.
Thes property will be use in server.xml of this bundle in Context section.
Signed-off-by: Vitalii Parfonov <vparfonov@codenvy.com>

### CODE REVIEW

**Comment by vparfonov** [comment]
@eivantsov @azatsarynnyy

File: assembly/assembly-wsagent-server/src/assembly/server.xml [hunk 1]
```diff
@@ -119,18 +119,13 @@
...
-        <!--Realm className="org.apache.catalina.realm.UserDatabaseRealm"
-               resourceName="UserDatabase"/-->
-      </Realm>
-
       <Host name="localhost"  appBase="webapps"
             unpackWARs="true" autoDeploy="false">
-        <Context docBase="/root/.apidocs/html" path="/artikdocs"/>
+
+        <!-- /root/.apidocs/html default location for Artik SDK docs here we will use  ${artik.docs.home} for make it more
+          flexible because folder with Artik Doc can have other location or doe's not exist -->
```
Comment (azatsarynnyy) [inline assembly/assembly-wsagent-server/src/assembly/server.xml]: typo
`doe's not` -> `doesn't`

Comment (vparfonov) [inline assembly/assembly-wsagent-server/src/assembly/server.xml]: thx

**Comment by azatsarynnyy** [comment]
Since `artik.docs.home` is a system property now
you can inject it's value as `@Named("sys.artik.docs.home") String docsPath` in `KeywordDocsService`.
And remove extra property `artik.apidocs.path` from the `che-machine-configuration.properties` [there](https://github.com/codenvy/artik-ide/blob/master/assembly/assembly-wsagent-war/src/main/webapp/WEB-INF/classes/codenvy/che-machine-configuration.properties#L59)

File: assembly/assembly-wsagent-server/src/assembly/setenv.sh [hunk 2]
```diff
@@ -21,11 +21,24 @@
 
 [ -z "${JPDA_ADDRESS}" ]  && JPDA_ADDRESS="4403"
 
+
+#Going to check is directory with Artik API Docs exist. If it exist set special property to the location of this folder
+#if not map it ro the webapps/ROOT. We need this workaround for staring WS-Agent in any case.
```
Comment (azatsarynnyy) [inline assembly/assembly-wsagent-server/src/assembly/setenv.sh]: typo
`map it ro` -> `map it to`

File: assembly/assembly-wsagent-server/src/assembly/setenv.sh [hunk 3]
```diff
@@ -21,11 +21,24 @@
...
+#Thes property will be use in server.xml of this bundle in Context section.
```
Comment (azatsarynnyy) [inline assembly/assembly-wsagent-server/src/assembly/setenv.sh]: **Thes** property **will be use** -> **This** property **is used**

**Pull request merged**